### DISCUSSION TITLE
Can You Ignore a Specific Business Object Property When Using the Csla Validator Tag

### DISCUSSION THREAD

**Discussion by MarkOverstreet** [MarimerLLC/csla#2013] [Questions]
I'm using the DevExpress TagBox Blazor (multiselect dropdown) control and it requires you to bind to an `IEnumerable<CustomType>` property which works for me until a validation occurs (using the  `<CslaValidator />` tag)

in which case the browser throws this exception ...

System.ArgumentException: Model does not implement required interface ICheckRules
   at Csla.Blazor.EditContextCslaExtensions.ValidateField(EditContext editContext, ValidationMessageStore messages, FieldIdentifier& fieldIdentifier)
   
I really just want to tell CSLA not to bother validating that one particular property on my business object.  Is there a way to do this?

Thanks
Mark

**Comment by rockfordlhotka** [comment]
I don't think the aspnetcore validation system has a way to ignore properties.

The `CslaValidator` throws that exception to "help" you, so you don't have any mystery about why certain bindings aren't validated. Perhaps that wasn't the right call? Maybe instead of an exception, it should write a message to the console or something?

**Comment by MarkOverstreet** [comment]
Yeah, maybe that would work.   If I could customize the behavior and normally get the exception but change it in certain cases that would work.   Or maybe there is a to capture the exception and eat it?

I really wish that DevExpress had not made their component require me to Cast my business object to IEnumerable<> and I don't think this would be a problem.

**Comment by Greg90277** [comment]
Has anyone found a resolution to this exception? I am encountering it with Blazor/Telerik controls.

**Reply by mikeclimbs23** [reply]
I'm in the same boat and I'm also using devexpress controls. Rocky may make a change so this doesn't raise exceptions.

Rocky and I discussed it over here [https://github.com/MarimerLLC/csla/discussions/2400]

**Comment by rockfordlhotka** [comment]
Added to backlog: https://github.com/MarimerLLC/csla/issues/2462
